tests/extra/fromEvent.ts

```diff
@@ -103,3 +136,56 @@ describe('fromEvent (extra)', () => {
     target.emit(2);
   });
 });
+
+describe('fromEvent (extra) - EventEmitter', () => {
+  it('should call addListener with expected parameters', () => {
+    const target = new FakeEventEmitter();
+    const stream = fromEvent(target, 'test');
+
+    stream.addListener({next: noop, error: noop, complete: noop});
+
+    assert.strictEqual('test', target.event);
```

Comment:
These are good tests. However over here there's some Yoda talk going on (e.g. "if monday is today", as opposed to "if today is monday"). The API is `assert.strictEqual(actual, expected)`, so it should be `assert.strictEqual(target.event, 'test')`